esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -809,7 +818,7 @@ bool EspHttpBinding::basicAuth(IEspContext* ctx)
 
     m_secmgr->updateSettings(*user,securitySettings, ctx->querySecureContext());
 
-    ctx->addTraceSummaryTimeStamp(LogMin, "basicAuth");
+    ctx->addTraceSummaryTimeStamp(LogMin, "basicAuth", TXSUMMARY_GRP_CORE);
```

Comment:
Point to ponder: With reuse of "enterprise" terminology, would the existing counterpart to the term (i.e., "community") be better or worse than "core"?

Reply:
core somehow feels a bit better to me. If you think "community" is better I'll switch though.

Reply:
I'm not going to push for a change.